.gitignore

```diff
@@ -48,3 +45,5 @@ versions.json
 /test-results.xml
 .temp-eslintcache
 /tests/fixtures/autofix-integration/temp.js
+yarn.lock
+package-lock.json
```

Comment:
These lock files should not be added to gitignore, since they are meant to be commited.
We can ignore the `yarn.lock` since we dont use it, but the ci server uses `package-lock.json` to determine if it needs to reinstall dependencies.

This change will break the CI Workflow.